scripts/run-with-strimzi.sh

```diff
@@ -37,17 +54,54 @@ if [ $MINIKUBE_MEM_EXIT -eq 0 ]; then
 else
   echo "no minikube memory configuration, defaulting to 4096M"
 fi
-INSTALL_DIR="sample-install"
 ${MINIKUBE} start "${MINIKUBE_CONF}"
+
+NAMESPACE=kafka
+
+# Prepare kustomize overlay
+cp -r "${SAMPLE_DIR}" "${KUSTOMIZE_TMP}"
+OVERLAY_DIR=$(find "${KUSTOMIZE_TMP}" -type d -name minikube)
+
+if [[ ! -d "${OVERLAY_DIR}" ]]; then
+     echo "$0: Cannot find minikube overlay within sample."
+     exit 1
+fi
+
+pushd "${OVERLAY_DIR}"
+${KUSTOMIZE} edit set namespace ${NAMESPACE}
+if [[ "${QUAY_ORG}" != 'kroxylicious' ]]; then
+  ${KUSTOMIZE} edit set image "quay.io/kroxylicious/kroxylicious=quay.io/${QUAY_ORG}/kroxylicious"
```

Comment:
Pointing out to @SamBarker that it is this line the overrides the image in the minikube overlay.